### PR TITLE
[cudaampper] main.cu uses namespace claragenomics::cudamapper

### PR DESCRIPTION
**Pull request by mimaric**
As discussed in PR #370 there is no reason not to have the content of `cudamapper/src/main.cu` in namespace `claragenomics::cudamapper::`

### CODE REVIEW

File: cudamapper/src/main.cu [hunk 1]
```diff
@@ -775,3 +775,13 @@ int main(int argc, char* argv[])
 
     return 0;
 }
+
+} // namespace cudamapper
+} // namespace claragenomics
+
+/// \brief main function
+/// main function cannot be in a namespace so using this function to call actual main function
+int main(int argc, char* argv[])
```
Comment (ahehn-nv) [inline cudamapper/src/main.cu:784]: Why not just leave main in the global namespace and do `using namespace claragenomcs::cudamapper;` inside?

Comment (ahehn-nv) [reply cudamapper/src/main.cu:784]: discussed offline.

**Comment by mimaric** [comment]
rerun tests

**Comment by mimaric** [comment]
rerun tests

**Pull request merged**